LICENCE.rst

```diff
@@ -1,7 +1,8 @@
 GrantNav
 ========
 
-Note: This license doesn't apply to the 360Giving theme and related assets (such as the logo).
+Note: This license doesn't apply to the 360Giving logo https://github.com/OpenDataServices/grantnav/blob/master/grantnav/frontend/static/images/360logo.png https://github.com/OpenDataServices/grantnav/blob/master/grantnav/frontend/static/images/360logo_invert.png
+The look-and-feel is that of 360Giving, and care should be taken when running public instance not to confuse users.
```

Comment:
"when running any public instance not to falsely give the impression, intentionally or otherwise, that the instance is operated by or related to 360Giving"?